lnwallet/test/test_interface.go

```diff
@@ -226,7 +226,8 @@ func assertTxInWallet(t *testing.T, w *lnwallet.LightningWallet,
 	// We'll fetch all of our transaction and go through each one until
 	// finding the expected transaction with its expected confirmation
 	// status.
-	txs, err := w.ListTransactionDetails(0, btcwallet.UnconfirmedHeight, "")
+	txs, err := w.ListTransactionDetails(0, btcwallet.UnconfirmedHeight,
```

Comment:
Please see our [formatting guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md) and existing code on how to format multi-line method/function calls.